lib/foreman_host_reports/engine.rb

```diff
@@ -48,6 +52,7 @@ class Engine < ::Rails::Engine
     config.to_prepare do
       Host::Managed.include ForemanHostReports::HostExtensions
       SmartProxy.include ForemanHostReports::HostExtensions
+      ::HostsController.include ForemanHostReports::Controller::HostsControllerExtensions
```

Comment:
I think you forgot to add this file to the commit:

`
Unable to autoload constant HostsControllerExtensions, expected /home/vagrant/foreman_discovery/app/controllers/foreman_discovery/concerns/hosts_controller_extensions.rb to define it (LoadError)
`

Reply:
Of course, pushed.